### ISSUE TITLE
Chapter 5 error reply showing up in history as ":unknown"

### ISSUE THREAD

**Issue by brentvukmer** [jepsen-io/maelstrom#30]
Is this because I'm populating `msg_id` in the reply body?

```
2023-01-10 10:06:45,020{GMT}    INFO    [n0 stderr] maelstrom.process: 2023/01/10 10:06:45 Failed to process txn request {c4 n0 {txn 35 [[append 6 5] [append 11 13] [append 5 2] [r 6 <nil>]]}}: [[append 6 5] [append 11 13] [append 5 2] [r 6 [1 2 3 4 5]]], RPC txn-conflict error (due to service response code 22)

2023-01-10 10:06:45,020{GMT}    INFO    [n0 stderr] maelstrom.process: 2023/01/10 10:06:45 Wrote error reply: {"src":"n0","dest":"c4","body":{"type":"error","msg_id":99,"in_reply_to":35,"code":30,"text":"CAS failed"}}

2023-01-10 10:06:45,022{GMT}    INFO    [jepsen worker 0] jepsen.util: 0        :info   :txn    [[:append 6 5] [:append 11 13] [:append 5 2] [:r 6 nil]]        [:unknown "CAS failed"]
```

**Comment by aphyr** [comment]
I don't think so. This may be a bug in that workload.

**Comment by aphyr** [comment]
Oh, I think this is fixed in 0.2.2!
